test/mockBot.js

```diff
@@ -31,13 +31,17 @@ module.exports = self = {
       chans: { '#test': null }
     }
 
-    const msg = {
-      body: text.substr(text.indexOf(' ') + 1),
-      args: text.substr(1).split(' '),
+    var msg = {
```

Comment:
```suggestion
    const msg = {
```

Reply:
`const` means I can't apply the `msg.cmds` fix later though.

Also the tests still don't work anyway because I'd have to reimplement all of `_commands.js`

Reply:
I think it does.
You can't overwrite a `const` object with an entirely new one, but you are able to modify object keys. ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#Description)

(`msg.cmds` is equivalent to `msg['cmds']`)